sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -1875,7 +1875,7 @@ def test_per_key_pipeline_checking_violated(self):
       (self.p
        | beam.Create(range(100)).with_output_types(int)
        | 'Num + 1' >> beam.Map(lambda x: x + 1).with_output_types(int)
-       | 'TopMod' >> combine.Top.PerKey(1, lambda a, b: a < b))
```

Comment:
Should we leave some of these tests for the remaining py2 code?

Reply:
Merging this as is. @robertwb please address this comment in a follow up.

Reply:
The choice of compare function here does not influence the type inference (which is the point of this test). Incidentally, these were also the default (natural ordering) so entirely redundant.